docs/zkapps/how-to-deploy-a-zkapp.mdx

```diff
@@ -1,6 +1,14 @@
 ---
 title: How to Deploy a zkApp
 hide_title: true
+description: A step-by-step guide on how to deploy a zkApp to the Berkeley Testnet. The guide covers configuring your smart contract project, requesting funds from the faucet, and deploying your smart contract.
+keywords:
+  - zkApp
+  - Deployment
+  - Berkeley Testnet
+  - Smart contract
+  - Configuration
```

Comment:
```suggestion
  - smart contract
  - configure a zkapp
```

Reply:
the few models that I've look at use lower case (probably keywords are case-insensitive?)